test/tools/OpenCover/OpenCover.psm1

```diff
@@ -0,0 +1,418 @@
+   Microsoft.PowerShell.Commands.Utility               62.38            0 64.54                0
+   Microsoft.PowerShell.ConsoleHost                    36.53            0 38.40                0
+   Microsoft.PowerShell.CoreCLR.AssemblyLoadContext    53.66            0 95.31                0
+   Microsoft.PowerShell.CoreCLR.Eventing               28.70            0 36.23                0
+   Microsoft.PowerShell.LocalAccounts                  73.15            0 84.32                0
+   Microsoft.PowerShell.PackageManagement              59.77            0 62.04                0
+   Microsoft.PowerShell.PSReadLine                     7.12             0 9.94                 0
+   Microsoft.PowerShell.Security                       15.17            0 18.16                0
+   Microsoft.WSMan.Management                          3.93             0 4.45                 0
+   Microsoft.WSMan.Runtime                             0                0 0                    0
+   powershell                                          100              0 100                  0
+   System.Management.Automation                        42.18            0 44.11                0
+#>
+function Compare-CodeCoverage
+{
+    [CmdletBinding()]
+    param ( 
+        [Parameter(Mandatory=$true,Position=0,ParameterSetName="file")][string]$RunFile1, 
+        [Parameter(Mandatory=$true,Position=1,ParameterSetName="file")][string]$RunFile2,
+        [Parameter(Mandatory=$true,Position=0,ParameterSetName="coverage")][Object]$Run1, 
+        [Parameter(Mandatory=$true,Position=1,ParameterSetName="coverage")][Object]$Run2
+        )
+
+    if ( $PSCmdlet.ParameterSetName -eq "file" ) 
+    {
+        [string]$xmlPath1 = (get-item $Run1File).Fullname        
+        $Run1 = (Get-CoverageData -xmlPath $xmlPath1)
+
+        [string]$xmlPath2 = (get-item $Run1File).Fullname
+        $Run2 = (Get-CoverageData -xmlPath $xmlPath2)
+    }
+    
+    (Get-CodeCoverageChange -r1 $Run1 -r2 $Run2)
+}
+
+## We are taking dependency on private build of OpenCover till a new release is available.
+## This is required for collecting code coverage numbers with portable PDBs.  
+<#
+.Synopsis
+   Install OpenCover by downloading the 4.6.589 version.  
+.Description
+   Install OpenCover version 4.6.589. This version supports debugType as portable.
+#>
+function Install-OpenCover
+{
+    param ( 
+        [parameter()][string]$version = "4.6.589",
+        [parameter()][string]$targetDirectory = $PWD,
+        [parameter()][switch]$force
+        )
+
+    $webclient = [System.Net.WebClient]::New()
+    $filename =  "opencover.${version}.zip"
+    $packageUrl = "https://ci.appveyor.com/api/buildjobs/xj78v6dac42uob8q/artifacts/main%2Fbin%2Fzip%2Fopencover.4.6.589.zip"
+    if ( test-path $PWD/$Filename ) 
+    {
+        if ( $force ) 
+        {
+            remove-item -force "$PWD/$Filename"
+        }
+        else 
+        {
+            throw "package already exists, not downloading"
```

Comment:
Added the path where the package was found to the error message. Made targetDirectory mandatory.